spec/latest/json-ld-framing/index.html

```diff
@@ -552,34 +543,109 @@
-    is extended to support <dfn data-lt="free-floating node">free-floating nodes</dfn>.
-    A <a>free-floating node</a> is a <a>node reference</a> which is not used as an <a>object</a>.</p>
-  <p>When run in the <strong>json-ld-framing-1.1-expand-frame</strong> processing mode, the algorithm is modified as follows:</p>
+<section id="frame-matching">
+  <h2>Frame Matching Algorithm</h2>
 
-  <ul>
-    <li>In step <strong>7.4.3</strong>: if <em>value</em> is a <a>dictionary</a> an
-      error has not been detected, and <em>expanded value</em> is set to the result of
-      performing the
-      <a href="../json-ld-api/#expansion-algorithm">Expansion Algorithm</a>
-      on <em>value</em> recursively, passing <a>active context</a>, <a>active property</a>,
-      and <em>value</em> for <em>element</em>.
-      <span class="ednote">dlongley: I think this is expanded, but not sure.</span></li>
-    <li>In step <strong>7.4.4</strong>: if <em>value</em> is an empty <a>dictionary</a> an
-      error has not been detected, and <em>expanded value</em> to <em>value</em>.</li>
-    <li>After step <strong>7.4.11</strong>: If <em>expanded property</em> is any other
-      framing keyword (<code>@explicit</code>, <code>@default</code>,
-      <code>@embed</code>, <code>@explicit</code>, <code>@omitDefault</code>, or
-      <code>@requireAll</code>),
-      set <em>expanded value</em> to the result of performing the <a
-      href="../json-ld-api/#expansion-algorithm">Expansion
-      Algorithm</a> recursively, passing <a>active context</a>, <a>active
-      property</a>, and <em>value</em> for <em>element</em>.</li>
-    <li>Skip step <strong>12</strong>: <a>free-floating nodes</a> are retained.</li>
-  </ul>
+  <p>The Frame Matching Algorithm is used as part of the <a href="#framing-algorithm">Framing algorithm</a>
+    to determine if a particular <a>node object</a> matches the critera set in a <a>frame</a>.
+    In general, a node object matches a frame if it meets the matches on <code>@type</code>,
+    or <code>@id</code>,
+    or if it matches given one of several different properties (or all properties, if the
+    <a>require all flag</a> is present.).</p>
+
+  <p class="note">As matching is performed on expanded node objects, all values will be in the form of an array.</p>
+
+  <p>Node matching uses a combination of JSON constructs to match <em>any</em>, <em>no</em>, or <em>some</em> specific values:</p>
+  <dl>
+    <dt><code>[]</code> (<code><dfn>match none</dfn></code>)</dt>
+    <dd>An empty array matches no values, or a value which is, itself, an empty array.</dd>
+    <dt><code>{}</code> (<code><dfn>wildcard</dfn></code>)</dt>
+    <dd>An array containing an empty object
+      (after excluding the framing properties <code>@default</code>, <code>@embed</code>, <code>@explicit</code>, <code>@omitDefault</code>, and <code>@requireAll</code>)
+      matches any value that is present, and does not match if there are no values.</dd>
```

Comment:
+1. I don't know if this is true or not yet, but it would be nice if we could simplify by saying "after excluding any framing keywords", rather than having to enumerate them.